### PR TITLE
Add chrome service

### PR DESCRIPTION
**Pull request by TOUFIKIzakarya**

### CODE REVIEW

File: Google Cloud/google-report/_meta/smart-descriptions.json
Comment (squioc) [inline Google Cloud/google-report/_meta/smart-descriptions.json:1]: Can you move these smartdescriptions upper? They are overridden by another one

File: Google Cloud/google-report/ingest/parser.yml [hunk 1]
```diff
@@ -278,3 +280,21 @@ stages:
...
           event.reason: '{% for param in json_event.message.events[0].parameters %}{% if param.name == "failure_type" %}{{param.value}}{% endif %}{% endfor %}'
         filter: '{{ json_event.message.events[0].name == "login_failure"}}'
+
+  set_chrome_fields:
+    actions:
+      - set:
+          event.category: ["web"]
+          organization.name: '{% for param in json_event.message.events[0].parameters %}{% if param.name == "ORG_UNIT_NAME" %}{{param.value}}{% endif %}{% endfor %}'
+          event.reason: '{% for param in json_event.message.events[0].parameters %}{% if param.name == "EVENT_REASON" %}{{param.value}}{% endif %}{% endfor %}'
+          device.id: '{% for param in json_event.message.events[0].parameters %}{% if param.name == "PRODUCT_ID" %}{{param.value}}{% endif %}{% endfor %}'
```
Comment (squioc) [inline Google Cloud/google-report/ingest/parser.yml:290]:
```suggestion
          device.model.identifier: '{% for param in json_event.message.events[0].parameters %}{% if param.name == "PRODUCT_ID" %}{{param.value}}{% endif %}{% endfor %}'
```

File: Google Cloud/google-report/ingest/parser.yml [hunk 2]
```diff
@@ -278,3 +280,21 @@ stages:
...
+          device.model.identifier: '{% for param in json_event.message.events[0].parameters %}{% if param.name == "DEVICE_NAME" %}{{param.value}}{% endif %}{% endfor %}'
```
Comment (squioc) [inline Google Cloud/google-report/ingest/parser.yml:291]:
```suggestion
          host.name: '{% for param in json_event.message.events[0].parameters %}{% if param.name == "DEVICE_NAME" %}{{param.value}}{% endif %}{% endfor %}'
```

File: Google Cloud/google-report/ingest/parser.yml
Comment (squioc) [inline Google Cloud/google-report/ingest/parser.yml:1]: I think you can also extract `VENDOR_NAME` into `device.manufacturer` and `DEVICE_PLATFORM` into `host.os.full`

**Review by squioc — CHANGES_REQUESTED** [review]
Thank you for the work. Some changes to apply.

File: Google Cloud/google-report/ingest/parser.yml
Comment (TOUFIKIzakarya) [reply Google Cloud/google-report/ingest/parser.yml:1]: Ok i will do that thanks

**Review by squioc — DISMISSED** [review]
LGTM

**Review by squioc — APPROVED** [review]
LGTM

**Pull request merged**